### PR TITLE
Dpcompass cals

### PR DESCRIPTION
**Pull request by wruef**
Adding calibrations and parsers for DP compass VEL3D spin tests

### CODE REVIEW

File: tools/cal_scripts/vel3d_cal_parser.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# VEL3DA calibration parser
+#
+# Create the necessary CI calibration ingest information from an VEL3DAA calibration file
```
Comment (mvardaro) [inline tools/cal_scripts/vel3d_cal_parser.py:5]: Should this just say "VEL3DA"?

**Review by mvardaro — COMMENTED** [review]
Looks good. My only comment is on some documentation of the VEL3DA parser (line 5).

File: tools/cal_scripts/vel3d_cal_parser.py (hunk 1 above)
Comment (wruef) [reply tools/cal_scripts/vel3d_cal_parser.py:5]: Yes..will fix in the next big push

**Review by mvardaro — APPROVED** [review]
Everything looks good except the one typo that will be fixed in the next push.

**Pull request merged**